ci-operator/config/openshift/azure-file-csi-driver-operator/openshift-azure-file-csi-driver-operator-main.yaml

```diff
@@ -50,12 +50,7 @@ tests:
   commands: make verify
   container:
     from: src
-- as: e2e-azure
```

Comment:
Azure File operator won't be installed here (it requires the feature gate), so there's no need to schedule this job.